### ISSUE TITLE
avail_circ_modes set to None causes OpenPMDTimeSeries.slider() to fail

### ISSUE THREAD

**Issue by jgerity** [openPMD/openPMD-viewer#118]
It appears that for Cartesian datasets, `avail_circ_modes` will be set to `None` in [params_reader.py](https://github.com/openPMD/openPMD-viewer/blob/dev/opmd_viewer/openpmd_timeseries/data_reader/params_reader.py#L99), but this appears to cause an error when calling `OpenPMDTimeSeries.slider()` for interactive exploration in an IPython (`4.2.1-aed0eae`) notebook with widgets.  This is replicated for the [example datasets](https://github.com/openPMD/openPMD-example-datasets)

Replacing `avail_circ_modes` with `[]` as needed after creating the `OpenPMDTimeSeries` object seems to yield the correct behavior:

```
import opmd_viewer

ts = opmd_viewer.OpenPMDTimeSeries('./example-3d/hdf5/')

%matplotlib inline
if ts.avail_circ_modes is None:
    ts.avail_circ_modes = []
ts.slider()
```

Unless I've misunderstood something about this object, using the empty list `[]` instead of `None` at [params_reader.py:L99](https://github.com/openPMD/openPMD-viewer/blob/dev/opmd_viewer/openpmd_timeseries/data_reader/params_reader.py#L99) will resolve the issue without the need for intervention on the user's part.

**Comment by RemiLehe** [comment]
Thanks a lot for pointing this out 😄  I think @ncook882 was observing the same bug during USPAS, but I forgot to follow up on this. 

In any case, I think the solution that you propose (in params_reader.py L99) is great. Would you be willing to do a pull request with this change (see [CONTRIBUTING.md](https://github.com/openPMD/openPMD-viewer/blob/master/CONTRIBUTING.md)), and put me as a reviewer? It may seem a little bit overkill to do a PR for just one line of code, but it allows us to nicely track issues and contributions to the code, so I would still go for it.

Thanks again!!

PS: Out of curiosity, which version of `ipywidgets` are you using? It seems that, depending on the version of this package, users may or may not observe this bug.

**Comment by jgerity** [comment]
Sure, I'll put in a PR with that change (and possibly moving [L270-272 from interactive.py](https://github.com/openPMD/openPMD-viewer/blob/dev/opmd_viewer/openpmd_timeseries/interactive.py#L270) up into the check at [L262 of the same](https://github.com/openPMD/openPMD-viewer/blob/dev/opmd_viewer/openpmd_timeseries/interactive.py#L262), if you agree), I wanted to open an issue first to make sure I hadn't misunderstood something about the way thetaMode works.

I'm using `ipywidgets` version `5.1.5`.

**Comment by RemiLehe** [comment]
Closed by PR #119
